lib/cfn-nag/ip_addr.rb

```diff
@@ -8,7 +8,7 @@ def ip4_localhost?(egress)
   end
 
   def ip6_localhost?(egress)
-    egress.cidrIpv6.is_a?(String) && egress.cidrIpv6 == '::1/128'
+    egress.cidrIpv6.is_a?(String) && (egress.cidrIpv6 == '::1/128' || egress.cidrIpv6 == ':1/128')
```

Comment:
Only other thing I would consider would be a test for this particular scenario of single vs. double colon.